app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js

```diff
@@ -269,6 +269,28 @@ angular.module( 'ngm.widget.project.details', [ 'ngm.provider' ])
 						$('#search_').focus();
 						$scope.search_input = $scope.search_input ? false : true;;
 					}
+
+					if ( $scope.project.definition.plan_component ) {
+						$scope.project.definition.hrp_plan = $scope.project.definition.plan_component.includes('hrp_plan');
+						$scope.project.definition.rmrp_plan = $scope.project.definition.plan_component.includes('rmrp_plan');
+						$scope.project.definition.interagencial_plan = $scope.project.definition.plan_component.includes('interagencial_plan');
+
+					 $scope.project.definition.humanitarian_component = $scope.project.definition.plan_component.includes('humanitarian_component');
+					$scope.project.definition.construccion_de_paz_component = $scope.project.definition.plan_component.includes('construccion_de_paz_component');
+					$scope.project.definition.desarrollo_sostenible_component = $scope.project.definition.plan_component.includes('desarrollo_sostenible_component');
+					$scope.project.definition.flujos_migratorios_component = $scope.project.definition.plan_component.includes('flujos_migratorios_component');
+
+					}/*else{
+						$scope.project.definition.hrp_plan = false;
+						$scope.project.definition.rmrp_plan = false;
+						$scope.project.definition.interagencial_plan = false;
+
+					 $scope.project.definition.humanitarian_component = false;
+					$scope.project.definition.construccion_de_paz_component = false;
+					$scope.project.definition.desarrollo_sostenible_component = false;
+					$scope.project.definition.flujos_migratorios_component = false;
+
+					}*/
```

Comment:
just comment for future but what if to loop through plan_component and assign to make it more general as plan functionality might be used for other countries